README.rst

```diff
@@ -1,4 +1,80 @@
 python-rapidjson
 ================
+Python wrapper around RapidJSON. RapidJSON_ is an extremely fast C++ json
+serialization library.
 
-Python wrapper around rapidjson
+We do not support legacy python versions, you will need to upgrade to Python 3
+to use this library.
+
+
+Getting Started
+---------------
+To get started using ``python-rapidjson`` you will need to install RapidJSON, on
+Ubuntu you do the following:
+
+.. code-block:: bash
+
+    $ sudo apt-get install rapidjson-dev
```

Comment:
I think this can be removed. My intent was to include the C++ source as part of this lib so it would just work. https://github.com/kenrobbins/python-rapidjson/tree/master/thirdparty/rapidjson/include/rapidjson